app/src/main/java/com/example/deejai/RoomSelection/RoomAdapter.kt

```diff
@@ -8,9 +11,11 @@ import androidx.cardview.widget.CardView
 import androidx.recyclerview.widget.RecyclerView.ViewHolder
 import androidx.recyclerview.widget.RecyclerView.Adapter
 import android.view.LayoutInflater
+import androidx.core.content.ContextCompat.startActivity
+import androidx.recyclerview.widget.RecyclerView
 import com.example.deejai.R
 
-class RoomAdapter(val rooms : List<Room>) : Adapter<RoomAdapter.RoomViewHolder>() {
+class RoomAdapter(private val rooms : List<Room>, private val c : Context) : Adapter<RoomAdapter.RoomViewHolder>() {
```

Comment:
Rename this context to something a bit more explanative (e.g.: context).

Reply:
done